src/html/locales/en/common.json

```diff
@@ -58,6 +68,37 @@
         "save": "Save",
         "release": "Ungrant",
         "approve": "Approve",
-        "decline": "Decline"
+        "decline": "Decline",
+        "finish": "OK"
+    },
+    "Authority": {
+        "label": "Authority",
+        "owner": "owner",
+        "editor": "write",
+        "viewer": "read"
+    },
+    "Sharing": {
+        "title": "Show different $t(glossary:label)",
+        "by": {
+            "label": "sharing my $t(glossary:label)"
+        },
+        "with": {
+            "label": "sharing with"
+        },
+        "Member": {
+            "label": "Sharing Member",
+            "add": "Add Member"
+        },
+        "Send": {
+            "label": "Send sharing request",
+            "description": "Send sharing request with the following authority.",
+            "btnFooter": "Send"
+        },
+        "Response": {
```

Comment:
Sharing.Received.label should it be "Received" instead of "Response"?

Reply:
"Received" is correct